source/supported-operations.txt

```diff
@@ -43,6 +43,19 @@ Comparison Functions and Operators
      - Return the largest argument
    * - ``IN()``
      - Check whether a value is within a set of values
+   * - ``INTERVAL(N,N1,N2,...)``
+     - Return 0-based index position of last number in the list to be ``< N``.
+       Return ``-1`` if ``N`` is ``NULL``
+     
+       ``N1``, ``N2``, etc. must be given in ascending order. 
+
+       .. example::
+
+          .. code-block:: sql
+
+             SELECT INTERVAL(26, 2, 9, 12, 28.5, 62, 2300);
+          
+          Returns ``3``
```

Comment:
Optional: maybe expand this to say why it returns 3, with something like:

> Returns ``3`` because ``12`` is the last number in the list less than ``26``.

Reply:
Agreed. Fixed.